src/components/TxSignSummary.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <tx-box-details :assets="tx.totalIncoming" :type="'success'">
```

Comment:
Let's use `v-if` to only display `tx-box-details` when it makes sense. If there is nothing is nothing to show, then we hide it.